docs/user-guide/docs-source/automated-scenario-generation.md

```diff
@@ -1,6 +1,6 @@
 # Automated Scenario Generation
 
-Ambit provides bulk scenario generation functionality. It does this by creating a batch of [individual SDFs](./individual-scenario-generation/) based on making permutations from each setting specified. Alongside those it also has a standalone file called a [Bulk Scenario Configuration (BSC)](#utilizing-bulk-scenario-generation-file) file that indicates how the batch was created. 
+Ambit provides bulk scenario generation functionality. It does this by creating a batch of [individual SDFs](./manual-scenario-generation) based on making permutations from each setting specified. Alongside those it also has a standalone file called a [Bulk Scenario Configuration (BSC)](#utilizing-bulk-scenario-generation-file) file that indicates how the batch was created. 
```

Comment:
This link is still broken with a WARNING 404. I think it should be `../manual-scenario-generation`.